src/components/forms/SelectOption/style.scss

```diff
@@ -6,7 +6,7 @@
   font-size: 14px;
 }
 
-.SelectOption--focused {
+.SelectOption--focused, .SelectOption:hover {
```

Comment:
Make a separate :hover style, maybe darker / transform scale(1.05). Needs to be visually distinguishable from fauxfocused element.